File: Java/opentraceability/src/main/java/opentraceability/OpenTraceabilityEventKDEProfile.java
package opentraceability;

public class OpenTraceabilityEventKDEProfile {
    public String XPath_V1 = "";
    public String XPath_V2 = "";
    public String JPath = "";

    public OpenTraceabilityEventKDEProfile() {}

    public OpenTraceabilityEventKDEProfile(String xPath_V1, String xPath_V2, String jPath) {
        XPath_V1 = xPath_V1;
        XPath_V2 = xPath_V2;
        JPath = jPath;
    }

    @Override
    public String toString() {
        return XPath_V1 + ":" + XPath_V2 + ":" + JPath;
    }
}

File: Java/OpenTraceability/src/main/opentraceability/opentraceabilityeventkdeprofile.kt
package opentraceability
class OpenTraceabilityEventKDEProfile {
    var XPath_V1: String = ""
    var XPath_V2: String = ""
    var JPath: String = ""

    constructor(xPath_V1: String,xPath_V2: String,jPath: String) {
        XPath_V1 = xPath_V1
        XPath_V2 = xPath_V2
        JPath = jPath
    }

    override fun toString(): String {
        return "$XPath_V1:$XPath_V2:$JPath"
    }
}
